src/main/features/directions-questionnaire/views/support-required.njk

```diff
@@ -0,0 +1,44 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, textInput, saveAndContinueButton, errorSummary, option, checkboxOption, textArea %}
+{% from "date.njk" import dateInput %}
+{% from "form.njk" import yesNoRadio %}
+
+{% set heading = t('Select any support you’d require for a court hearing (optional)') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form method="post" class="analytics-submit-event-trigger" data-event-action="DQ: Support Required" novalidate>
+        {{ csrfProtection(csrf) }}
+        {% set error = form.hasErrors() %}
+        <fieldset id="supportRequired" class="form-group {% if error %} form-group-error {% endif %}">
+          <legend>
+            <span class="visuallyhidden">
+              {{ t('Select any support you’d require for a court hearing (optional)') }}
+            </span>
+          </legend>
+          {% call expandableCheckboxOption(form = form, name = 'languageSelected', value = 'true', label = "Language Interpreter", checked = form.valueFor('languageSelected')) %}
+            {{ textInput(label = t('What language needs to be interpreted?'), hint = t('You’ll have to arrange and pay for interpreters except for Welsh.'), form = form, name = 'languageInterpreted', inputClass = '') }}
+          {% endcall %}
+          {% call expandableCheckboxOption(form = form, name = 'signLanguageSelected', value = 'true', label = t('Sign Language Interpreter'), checked = form.valueFor('signLanguageSelected')) %}
+            {{ textInput(label = t('Enter the sign language you need'), form = form, name = 'signLanguageInterpreted', inputClass = '') }}
+          {% endcall %}
+          {{ option(form = form, name = 'hearingLoopSelected', value = 'true', type = 'checkbox', checked = form.valueFor('hearingLoopSelected'), label = t('Hearing Loop')) }}
+          {{ option(form = form, name = 'disabledAccessSelected', value = 'true', type = 'checkbox', checked = form.valueFor('disabledAccessSelected'), label = t('Disabled Access')) }}
+          {% call expandableCheckboxOption(form = form, name = 'otherSupportSelected', value = 'true', label = 'Other support', checked = form.valueFor('otherSupportSelected')) %}
+            {{ textArea(label = t('Explain the other support you need at your hearing'), form = form, name = 'otherSupport') }}
+          {% endcall %}
+        </fieldset>
+        {{ saveAndContinueButton() }}
+      </form>
+    </div>
+  </div>
+{% endblock %}
+
+{% macro expandableCheckboxOption(form, name, value, checked = false, label, hint = '', bold = false, additionalClasses = undefined) %}
```

Comment:
there are now 4 of these across different pages. can we extract them out to a common place?

Reply:
I'll have a look at getting it into form.njk in the common-frontend project

Reply:
put it in the common area of the citizen frontend. we need to migrate away from the common-frontend at some point as we will not be doing any more development on legal so it has become redundant keeping everything as a separate repo